jenkins-master/jobs/scripts/workspace/config.py

```diff
@@ -18,11 +20,13 @@
                 'tier': 3,
             },
             'logs': {
```

Comment:
Something for discussion here... Given that we do not only upload logs, maybe we want to name that `artifacts`?